manifests/init.pp

```diff
@@ -56,14 +56,7 @@
   String  $config_file_owner                     = 'root',
   String  $config_file_group                     = 'root',
   Stdlib::Filemode $config_file_mode             = '0644',
-  Hash    $receivers                             = {
-    'otlp' => {
-      'protocols' => {
-        'http' => {},
-        'grpc' => {},
-      },
-    },
-  },
+  Optional[Hash] $receivers                      = undef,
```

Comment:
As suggested elsewhere: you can use
```suggestion
  Hash $receivers = {},
```

Or even tighter (since you're later checking it anyway)
```suggestion
  Hash[String, Hash] $receivers = {},
```